_topic_map.yml

```diff
@@ -620,18 +620,29 @@ Topics:
         File: ruby_on_rails
       - Name: Setting Up a Nexus Mirror
         File: maven_tutorial
-  - Name: Opening a Remote Shell to Containers
-    File: ssh_environment
-  - Name: Templates
-    File: templates
-  - Name: Service Accounts
-    File: service_accounts
   - Name: Builds
-    File: builds
-  - Name: Managing Images
-    File: managing_images
-  - Name: Quotas and Limit Ranges
-    File: compute_resources
+    Dir: builds
+    Topics:
+      - Name: How Builds Work
+        File: index
+      - Name: Basic Build Operations
+        File: basic_build_operations
+      - Name: Build Inputs
+        File: build_inputs
+      - Name: Build Output
+        File: build_output
```

Comment:
i'm with @ahardin-rh.  

1) calling it "source" is misleading, since we sometimes encourage people to do builds with things that aren't classically considered "source"
2) someday we may support other outputs (or add a section explaining how to do other outputs) that aren't images, so pushing to a registry, while accurate today, paints us into a corner for expanding it later w/o doing another refactor/rename.